config/donation-settings.js

```diff
@@ -39,6 +39,30 @@ const donationSubscriptionConfig = {
   }
 };
 
+// Shared paypal configuration
+const paypalConfig = {
+  production: {
+    webhookId: '',
+    durationPlans: {
+      month: {
+        '500': {
+          planId: ''
+        }
+      }
+    }
+  },
```

Comment:
Yes, since they are going to be long list of plan ids, we decided to keep them all in the donation-settings and display the right one based on the env variable and the donation amount.

Reply:
If I understand this correctly, `paypalConfig.production` is what gets used, but that doesn't (currently) have a `webhookId`.  So, is that property going to be overwritten by environment variables?

Reply:
no I just don't have that yet :). I will just add it when we get everything setup at the end.